pkg/operator/ceph/cluster/cluster.go

```diff
@@ -168,7 +168,7 @@ func (c *ClusterController) initializeCluster(cluster *cluster) error {
 
 	clusterInfo, _, _, err := mon.LoadClusterInfo(c.context, c.OpManagerCtx, cluster.Namespace)
 	if err != nil {
-		logger.Infof("clusterInfo not yet found, must be a new cluster")
+		logger.Infof("clusterInfo not yet found, must be a new cluster. %v", err)
```

Comment:
```suggestion
		logger.Errorf("clusterInfo not yet found, must be a new cluster. %v", err)
```
?

Reply:
It's not an error perse, so info is fine since it's likely a new cluster.

Reply:
Actually, this seems like a problem that we are always printing the error and continuing. If we know 100% that the failure is because it's a new cluster, then we can continue. But if it's a transient k8s api error, we don't want to continue and we need to fail the reconcile.

Reply:
Let's check for IsNotFound.